daft/daft/__init__.pyi

```diff
@@ -1177,22 +1177,6 @@ def get_udf_names(expression: PyExpr) -> list[str]: ...
 def resolve_expr(expr: PyExpr, schema: PySchema) -> tuple[PyExpr, PyField]: ...
 def hash(expr: PyExpr, seed: Any | None = None) -> PyExpr: ...
 def cosine_distance(expr: PyExpr, other: PyExpr) -> PyExpr: ...
-def url_download(
-    expr: PyExpr,
-    max_connections: int,
-    raise_error_on_failure: bool,
-    multi_thread: bool,
-    config: IOConfig,
-) -> PyExpr: ...
-def url_upload(
-    expr: PyExpr,
-    folder_location: PyExpr,
-    max_connections: int,
-    raise_error_on_failure: bool,
-    multi_thread: bool,
-    is_single_folder: bool,
-    io_config: IOConfig | None,
-) -> PyExpr: ...
```

Comment:
These are not used anywhere, removing them so that we can remove the original `UrlUploadArgs` and `UrlDownloadArgs`